ucm2/codecs/wsa884x/TwoSpeakerSeq.conf

```diff
@@ -0,0 +1,27 @@
+EnableSequence [
+	cset "name='SpkrLeft COMP Switch' 1"
+	cset "name='SpkrLeft BOOST Switch' 1"
+	cset "name='SpkrLeft DAC Switch' 1"
+	cset "name='SpkrLeft PBR Switch' 1"
+	cset "name='SpkrLeft VISENSE Switch' 0"
+	cset "name='SpkrLeft WSA MODE' 0"
+	cset "name='SpkrRight COMP Switch' 1"
+	cset "name='SpkrRight BOOST Switch' 1"
+	cset "name='SpkrRight DAC Switch' 1"
+	cset "name='SpkrRight PBR Switch' 1"
+	cset "name='SpkrRight VISENSE Switch' 0"
+	cset "name='SpkrRight WSA MODE' 0"
+]
```

Comment:
I think `cset "name='SpkrLeft PA Volume' 12"` and `cset "name='SpkrRight PA Volume' 12"` are missing in the EnableSequence